### ISSUE TITLE
Possible to access the probability matrix?

### ISSUE THREAD

**Issue by astridfilt** [robert-graf/VIBESegmentator#7]
I was wondering if it was somehow possible to get the probability matrix as output for a chosen segmentation, e.g. the liver?

**Comment by robert-graf** [comment]
Yes, the "run_inference_on_file" function has a boolean flag "logits" to return the logits. I only used it at the beginning of the project, so there may be some regression. Just follow the flag to the root if you encounter an issue. 

You have to change the code yourself to save them.
